### PR TITLE
update operator-sdk to use local/bin

### PR DESCRIPTION
**Pull request by yazug**
specify version for operator-sdk for make bundle
download operator-sdk pinned version when generating bundle update make bundle to use local operator-sdk

### CODE REVIEW

**Review by dprince — APPROVED** [review]

**Comment by dprince** [comment]
recheck

**Comment by dprince** [comment]
recheck

File: Makefile [hunk 1]
```diff
@@ -233,13 +241,30 @@ ginkgo: $(GINKGO) ## Download ginkgo locally if necessary.
...
+	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
+	curl -sSLo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_$${OS}_$${ARCH} ;\
+	chmod +x $(OPERATOR_SDK) ;\
+	}
+else
+OPERATOR_SDK = $(shell which operator-sdk)
+endif
+endif
+
 .PHONY: bundle
```
Comment (dprince) [inline Makefile:261]: the bundle-cache-extra-data here is needed

**Review by dprince — CHANGES_REQUESTED** [review]
@yazug The CI failures here are valid. The updated 'bundle' target needs to have bundle-cache-extra-data re-added to fix it

**Comment by yazug** [comment]
recheck

**Comment by abays** [comment]
recheck

**Pull request merged**